### PR TITLE
Vvv custom comment example

### PR DESCRIPTION
**Pull request by tomjn**
Adds a commented out example to the `vvv-config.yml` file, demonstrating adding a new site, as well as a reminder to copy the file to `vvv-custom.yml`

### CODE REVIEW

File: vvv-config.yml [hunk 1]
```diff
@@ -11,6 +11,15 @@ sites:
...
       - build.wordpress-develop.dev
 
+  # This would create a standard WordPress site in www/example-site available
+  # at http;//my-example-site.dev
+  # Remember to copy this file to vvv-custom.yml before making changes
+
+  #example-site:
+  #  repo: https://github.com/Varying-Vagrant-Vagrants/vvv-wordpress-default.git
+  #  hosts:
+  #    - my-example-site.dev
```
Comment (LoreleiAurora) [inline vvv-config.yml]: This example would still use the wordpress_default db.

Comment (tomjn) [inline vvv-config.yml]: do we have a generic WP install that can replace it? Similar to https://github.com/tomjn/vvv-wp-multisite-subdir ?

Comment (LoreleiAurora) [inline vvv-config.yml]: Ill put one together :)

Comment (LoreleiAurora) [inline vvv-config.yml]: https://github.com/Varying-Vagrant-Vagrants/custom-site-template

**Pull request merged**